addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -674,6 +674,19 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
     _onContextGet(event) {
         return event.data.callback(this._context);
     }
+    /**
+     * Returns the website service context.
+     *
+     * @param event
+     * @returns {*}
+     * @private
+     */
+    _onServiceContextGet(event) {
+        const readOnly = {
+            isMobile: this.websiteService.context.isMobile,
+        };
+        return event.data.callback(readOnly);
+    }
```

Comment:
```suggestion
    /**
     * Retrieves the website service context.
     *
     * @private
     * @param {OdooEvent} ev
     */
    _onServiceContextGet(ev) {
        ev.data.callback({
            isMobile: this.websiteService.context.isMobile,
        });
    }
```

- `@private` first
- Missing type for the param definition
- The handler should not "return" something (detail but also used "retrieves" in the text)
- not using "event" but "ev" instead to avoid problems with the event global variable of Chrome
- Removing temporary variable